docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -45,108 +45,15 @@ The example is not Stake Wars specific, the operator is responsible for finding
-You can adjust the `--keystore-path` if you prefer to generate the keys in a different location.  
-:::
-
-2. Open another terminal and use the `curl` command to make an **RPC** call to your node. This will generate a new key in the `--keystore-path` location you specified earlier. 
-
-<Tabs groupId="OS">
-
-<TabItem value="windows" label="🖼️ Windows" default>
-
-<CodeBlock>
-{`curl.exe -X POST http://127.0.0.1:9945 \`
--H "Content-Type: application/json;charset=utf-8" \`
--d '{
-"jsonrpc":"2.0",
-"id":1,
-"method":"author_rotateKeys",
-"params": []
-}'`}
-</CodeBlock>
-</TabItem>
-
-
-<TabItem value="macos" label="🍎 macOS">
-<CodeBlock>
-{`curl -X POST http://127.0.0.1:9945 \\
--H "Content-Type: application/json;charset=utf-8" \\
--d '{
-"jsonrpc":"2.0",
-"id":1,
-"method":"author_rotateKeys",
-"params": []
-}'`}
-</CodeBlock> 
-</TabItem>
-
-<TabItem value="linux" label="🐧 Ubuntu">
-
-<CodeBlock>
-{`curl -X POST http://127.0.0.1:9945 \\
--H "Content-Type: application/json;charset=utf-8" \\
--d '{
-"jsonrpc":"2.0",
-"id":1,
-"method":"author_rotateKeys",
-"params": []
-}'`}
-</CodeBlock>
-</TabItem>
-
-</Tabs>
-</TabItem>
-
-<TabItem value="docker" label="UI (using Docker)">
-
-1. Make sure to have [Docker installed](https://www.docker.com/get-started/).
-You can run `docker -v` in the terminal of your choice to make sure it's installed. 
-2. Start a developer node by running
-`./target/release/subspace-node --chain dev -- --domain-id 0 --keystore-path tmp/keystore --rpc-cors all`
-
-:::tip
-You can use any chain to generate a keypair, we recommend using a `dev` chain for simplicity. 
-You can adjust the `--keystore-path` if you prefer to generate the keys in a different location.  
-:::
-
-3. Start a local polkadot interface portal by running a docker contrainer.
-
-`docker run --rm -it --name polkadot-ui -e WS_URL=ws://0.0.0.0:9945 -p 80:80 jacogr/polkadot-js-apps:latest`
-
-4. Proceed to the browswer and type `localhost` in the search bar. You should be taken to the polkadot portal. 
-
-![RPC-2](/img/doc-imgs/operators-staking/RPC-2.png)
-
-5. Navigate to `developer` -> `rpc calls`
-
-6. Select `author` in `call the selected endpoint` and pick a `rotateKeys()` in the dropdown.
-
-![RPC-3](/img/doc-imgs/operators-staking/RPC-3.png)
-
-7. Press on `Submit RPC call`.
-
-8. You will see a newly generated key on the screen. The key will also be written in a `keystore` location you specified earlier. 
-
-:::tip
-You will use this key in order to register an operator later.
-:::
-
-</TabItem>
-
-</Tabs>
+Make sure to use the base **--base-path** for the key generation, and running an operator node and make sure to replace **--domain-id value** with the domainID you would like to be an operator on.
 
 **You have successfully generated an operator key, congratulations!**
 
+The keys were generated in the subfolder of **PATH_TO_NODE** under `/domains/domainID/keystore`. If you don't see the generated keypair in the subfolder, something went wrong. 
```

Comment:
Should we include a warning about how important it is that users record the seed phrase output to the CLI?

Reply:
Probably, though CLI will already print a warning on its own. Also you can always rotate the key, it is just important to not leak it or else someone can get operator slashed.

Reply:
Added a short note below.